R/get_exons_tracks_gtf.R

```diff
@@ -0,0 +1,93 @@
+#' Function gather exon locations per gene to plot and annotate breakpoints
+
+#' @param genome human genome version (hg38 and hg19 supported)
+#' @param gtf_path gencode gtf file path,  gencode.v27.primary_assembly.annotation.gtf.gz can be downloaded from "ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_27/gencode.v27.primary_assembly.annotation.gtf.gz"
+#'
+#' @export
+#'
+#' @return exon location dataframe
+#'
+#' @examples
+#' library(EnsDb.Hsapiens.v86)
+#'
+#' \dontrun{
+#' # using hg38 gencode gtf to get exon tracks
+#' download.file("ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_27/gencode.v27.primary_assembly.annotation.gtf.gz", "gencode.v27.primary_assembly.annotation.gtf.gz")
```

Comment:
I think we could and should use AnnotationHub for this, rather than downloading the gtf from the web.
Again, it is also for the benefits of having these things with a caching mechanism.